docs/excel/excel-add-ins-error-handling.md

```diff
@@ -47,12 +47,15 @@ When an Excel JavaScript API request fails to run successfully, the API returns
 
 The following table is a list of errors that the API may return.
 
-|error.code | error.message |
+|error.code | Error Message |
```

Comment:
Some table header formatting suggestions: 
```suggestion
|`error.code` | Error message |
```
This makes it clear the `error.code` is an actual property (and, by extension, shows "Error message" is not a code property). It also follows the guidelines for header capitalization (only the first word and proper nouns).